keyboards/dztech/volcano660/volcano660.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
I have a feeling this board can also support the `66_ansi` and `66_iso` community layouts.